src/Pages/hyojeunglee/Main/Main.js

```diff
@@ -0,0 +1,22 @@
+import React, { Component } from "react";
+import Nav from "../../../Components/Nav/Nav";
+import Article from "./Article";
+import Aside from "./Aside";
+import "../../../Styles/common.scss";
+import "./Main.scss";
+
+class Main extends Component {
+  render() {
+    return (
+      <div className="Main">
+        <Nav />
+        <div className="mainBox">
+          <Article />
+          <Aside />
+        </div>
+      </div>
```

Comment:
```suggestion
      <>
        <Nav />
        <div className="Main">
          <Article />
          <Aside />
        </div>
      </>
```

- 이렇게도 많이 사용합니다. 참고해주세요.

Reply:
아앗 이부분 해결해주셔서 감사합니다! 어떻게 써야할지 고민했었어요!